cmd/operator/main.go

```diff
@@ -256,3 +280,101 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+// +kubebuilder:rbac:groups=admissionregistration.k8s.io,resources=validatingwebhookconfigurations,verbs=get;list;watch;update;patch
+// +kubebuilder:rbac:groups="",namespace=keda,resources=secrets,verbs=get;list;watch;create;update;patch;delete
```

Comment:
this should be probably deleted, right?

Reply:
no, this is required in the operator because it generates the certificates and stores them as secret in keda namespace. I plan to deploy this conditionally in helm for cases where end-users manage the certs externally

Reply:
and this it have to be here? wouldn't be webhook a better location?

Reply:
no, because it's the operator who does the job xD I mean, even a user doesn't use the webhooks, the operator still generates the certificates (because they will be used for the other communications). The operator needs access to secrets in keda namespace

Reply:
What I am trying to say is that these markers a little bit "lost" in this file.

Reply:
oh, good point! I have placed it there because it is closest to the code where it's needed, when I move this to its own package I'll move them there also

Reply:
yeah :)